components/__tests__/__snapshots__/HomeScreen-test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `null`;
```

Comment:
Are these functioning and if not is there a reason they are being added?

Reply:
have removed both HomeScreen-test files and re-pushed.